scripts/test/watch

```diff
@@ -1,2 +1,3 @@
 #!/bin/sh
-exec filewatcher -L 6 -x build -x script go test -timeout 10s -v './${dir}'
+# shellcheck disable=SC2154
+exec filewatcher -L 6 -x build -x script go test -timeout 10s -v "./${dir}"
```

Comment:
This is wrong. It needs to be single quotes around `./${dir}` that literal is passed to `filewatcher` and interpolated by it. I realize now that choosing bash-like variables for this tool was not a great option, but I haven't had a chance to change it yet.

Does the shellcheck whitelist need to exist with single quotes?